tests/test_flask_util.py

```diff
@@ -282,15 +285,18 @@ def test_callback_view_errors(self):
 
         # Code exchange error
         with self.app.test_client() as c:
-            with Http2Mock(status=500):
+            with self.app.test_request_context():
```

Comment:
These ten lines look token-for-token identical to the ten lines fifty lines earlier. Pull out a common helper function?